src/vs/workbench/contrib/inlineChat/browser/inlineChatController.ts

```diff
@@ -456,6 +459,8 @@ export class InlineChatController implements IEditorContribution {
 		} else if (options.isUnstashed) {
 			delete options.isUnstashed;
 			return State.SHOW_RESPONSE;
+		} else if (this._session.chatModel.hasRequests && this._session.chatModel.requestInProgress) {
```

Comment:
nit but `this._session.chatModel.requestInProgress` actually implies `hasRequest`